setup.py

```diff
@@ -24,6 +24,7 @@
     "pytest-cache>=1.0",
     "pytest-cov>=1.8.0",
     "pytest>=2.8.0",
+    "mock==3.0.5",
```

Comment:
We should be also able to say here:

```
"mock>=3.0,<4.0",
```

which will open a possibility for some bug fix releases perhaps.

Reply:
Done 👍